crates/holochain/src/lib.rs

```diff
@@ -5,6 +5,15 @@
 // We have a lot of usages of type aliases to `&String`, which clippy objects to.
 #![allow(clippy::ptr_arg)]
 
+#[cfg(feature = "hdk")]
+pub use hdk::HDI_VERSION;
+
+#[cfg(feature = "hdk")]
+pub use hdk::HDK_VERSION;
+
+/// Current Holochain Conductor rust crate version.
+pub const HC_VERSION: &str = env!("CARGO_PKG_VERSION");
```

Comment:
```suggestion
pub const HOLOCHAIN_VERSION: &str = env!("CARGO_PKG_VERSION");
```

i find `HC` is ambiguous as there's a binary (crate) named `hc`